### PR TITLE
release 3.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
@alexjfisher any idea why the entry for 3.0.0 contains way to many issues and PRs?

File: HISTORY.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
+## [v2.2.0](https://github.com/voxpupuli/puppet-staging/tree/v2.2.0) (2017-02-11)
```
Comment (alexjfisher) [inline HISTORY.md]: The HISTORY.md is supposed to contain what *was* in CHANGELOG.md before running gcg...

**Comment by bastelfreak** [comment]
@alexjfisher I reviewed the existing CHANGELOG.md in master. Many releases are missing there. This HISTORY.md adds also infos about the missing releases which is in my opinion good. I tagged the old modulesync PRs so they don't show up anymore in the changelog. Should we keep the current one in this PR or should I put the CHANGELOG.md from master into HISTORY.md? I prefer the current changelog in this PR, it is more detailed and nothing is missing.

**Comment by bastelfreak** [comment]
ping @alexjfisher

**Pull request merged**